exporter/datadogexporter/config.go

```diff
@@ -494,6 +392,91 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+var _ error = (*renameError)(nil)
+
+// renameError is an error related to a renamed setting.
+type renameError struct {
+	// oldName of the configuration option.
+	oldName string
+	// newName of the configuration option.
+	newName string
+	// oldRemovedIn is the version where the old config option will be removed.
+	oldRemovedIn string
+	// issueNumber on opentelemetry-collector-contrib for tracking
+	issueNumber uint
+}
+
+// List of settings that have been removed, but for which we keep a custom error.
+var removedSettings = []renameError{
+	{
+		oldName:      "metrics::send_monotonic_counter",
+		newName:      "metrics::sums::cumulative_monotonic_mode",
+		oldRemovedIn: "v0.50.0",
+		issueNumber:  8489,
+	},
+	{
+		oldName:      "tags",
+		newName:      "host_metadata::tags",
+		oldRemovedIn: "v0.52.0",
+		issueNumber:  9099,
+	},
+	{
+		oldName:      "send_metadata",
+		newName:      "host_metadata::enabled",
+		oldRemovedIn: "v0.52.0",
+		issueNumber:  9099,
+	},
+	{
+		oldName:      "use_resource_metadata",
+		newName:      "host_metadata::hostname_source",
+		oldRemovedIn: "v0.52.0",
+		issueNumber:  9099,
+	},
+	{
+		oldName:      "metrics::report_quantiles",
+		newName:      "metrics::summaries::mode",
+		oldRemovedIn: "v0.53.0",
+		issueNumber:  8845,
+	},
+	{
+		oldName:      "metrics::instrumentation_library_metadata_as_tags",
+		newName:      "metrics::instrumentation_scope_as_tags",
+		oldRemovedIn: "v0.56.0",
+		issueNumber:  11135,
+	},
+}
+
+// Error implements the error interface.
+func (e renameError) Error() string {
+	return fmt.Sprintf(
+		"%q has been deprecated in favor of %q and will be removed in %s or later. See github.com/open-telemetry/opentelemetry-collector-contrib/issues/%d",
```

Comment:
```suggestion
		"%q has been deprecated in favor of %q and will be removed in %s or later. See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/%d",
```
This will make it easier to open in the terminal, by double-clicking and holding the Cmd key on Mac, as an example.